.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and test
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Can you add a Windows environment to test as it had AppVeyor CI for Windows? It can be done with `matrix` like : https://github.com/embulk/embulk-standards/blob/v0.10.36/.github/workflows/check.yml#L11-L15